docker-compose-full.yml

```diff
@@ -56,6 +57,9 @@ services:
     volumes:
       - ../v3-website:/usr/src/app
       - /usr/src/app/node_modules
+      - type: volume
+        source: tmp
+        target: /tmp
```

Comment:
This volume mapping is required for the tooling invoker to work correctly with the new file-based approach of passing solutions between the website and tooling-invoker.